demo/src/layouts/index.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import MainLayout from './MainLayout';
 
-const Layout = (props) => <MainLayout {...props} />;
+const Layout = (props) => {
+  const { children, pageContext } = props;
+
+  switch (pageContext.layout) {
+    case 'basic':
+      return children;
+    default:
+      return <MainLayout {...props} />;
+  }
+};
+
+Layout.propTypes = {
+  children: PropTypes.node,
+  pageContext: PropTypes.object.isRequired,
```

Comment:
Do we want to add a shape for this that includes `layout`?

Reply:
Since this is the demo site, I wasn't too careful on proper prop types 😄 . I'd be happy to add it if you feel strongly about it.

Reply:
I feel like I go on autopilot and forget to look at the filepath. Totally fine in the demo 👍

Reply:
No problem!